packages/ui/src/form/styles.css.ts

```diff
@@ -3,11 +3,23 @@ import { globalStyle, style } from '@vanilla-extract/css';
 
 const errorColor = '#FC5555';
 
-export const formField = style({
+export const formItem = style({
   position: 'relative',
   width: '100%',
+  marginTop: rem(8),
+
+  selectors: {
+    '&:first-of-type': {
+      marginTop: 0,
+    },
+  },
 });
 
+// export const formField = style({
+//   position: 'relative',
+//   width: '100%',
+// });
+
 export const control = style({});
 
 export const label = style({
```

Comment:
```suggestion
export const label = style({
  fontWeight: 500
```

레이블과 인풋 모두 `fontWeight`가 400으로 되어 있어서 500으로 변경이 필요해요.

Reply:
아하 medium이 400의 굵기라고 착각해서 이렇게 했던 것 같네요ㅠㅠ 수정하겠습니다!